addons/event/views/event_views.xml

```diff
@@ -587,21 +587,32 @@
                     <field name="event_ticket_id"/>
                     <templates>
                         <t t-name="kanban-box">
-                            <div class="oe_kanban_card oe_kanban_global_click">
+                            <div class="oe_kanban_card oe_kanban_global_click" style="width:400px">
```

Comment:
2 things here:

- We try to avoid inline-style, either we find a suitable bootstrap class, either we create a custom class and add the styling in scss
- I don't think it's a good idea to hardcode the width of cards, we should let the web framework handle the width optimally

Reply:
I wanted to make it look like the screenshot, do you think we need to create a specific responsive case or the default behavior is enough...?

Reply:
Try to keep it as the default size and check that everything is correctly fitting / displayed.
I've also suggested to LNA that big icons are not so sexy in desktop view so we will probably rework that part too, which will affect the size as well.

I would say remove the width style for now and we will check that last.

Reply:
Alright, I put it back the default behavior.